### ISSUE TITLE
Unable to set pitch trim when running consecutive flight.

### ISSUE THREAD

**Issue by ecortinas** [flybywiresim/a32nx#7396]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2022-08-02T16:30:10+00:00",
    "ref": "refs/heads/master",
    "sha": "09faacef1c06559441e4f1f88c9627b5bb3a7079",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "master:09faacef"
}
```

### Describe the bug

When I finish a flight and try to run another one, when I try to adjust the pitch, it automatically rotates itself back to 0 no matter what I do.

### Expected behavior

Pitch should allow adjustment when starting another flight.

### Steps to reproduce

1. Perform a full flight
2. Try to adjust pitch trim on new consecutive flight where you don't end the session when going into next flight.

**Comment by ecortinas** [comment]
https://imgur.com/a/efB0hOZ

**Comment by tracernz** [comment]
You have hydraulics at this point (e.g. engines running)?

**Comment by ecortinas** [comment]
Yup, already pushed back and ready to taxi.

**Comment by crocket63** [comment]
This is a known issue with the FBW state machine that locks ground mode after landing.

Already solved in experimental, will be definitly solved when #6913 is merged
